### PR TITLE
[Bug fix] SageMaker case sensitivity

### PR DESCRIPTION
**Pull request by bnsblue**
# TL;DR
A lot of the fields in SageMaker CRDs are case sensitive, but only some of them are validated at CRD level. The bug this PR fixed was detected by the operator.

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
A lot of the fields in SageMaker CRDs are case sensitive, but only some of them are validated at CRD level. The bug this PR fixed was detected by the operator. We use `strings.Title(strings.ToLower(<protoc enum>.String()))` to fix it.

## Follow-up issue
_NA_

### CODE REVIEW

**Comment by kumare3** [comment]
Have we tested sufficient surface area, that this is not an issue with other enums? All Protobuf enums are capitalized, i would presume all of them would have a problem? no?

**Comment by kumare3** [comment]
👍

**Comment by bnsblue** [comment]
@kumare3 I’ve double checked that, except for the “InputContentType” enum which requires a explicit translation, all the other enums that I use in SageMaker plugin are wrapped inside the same strings.Title(strings.ToLower()) thing
